packages/params/src/gas-price-oracles/oracles/base.ts

```diff
@@ -0,0 +1,7 @@
+import { IGetGasFeeResult, IOracle } from "./interfaces";
+import { getEtherscanGasFee } from "./utils";
+
+export const getBaseGasFee: IOracle = (
+  apiKey: string | undefined
+): Promise<IGetGasFeeResult> =>
+  getEtherscanGasFee("https://api.basescan.org/api", apiKey);
```

Comment:
Does this work? I thought this too was returning incorrect values?

Reply:
checked these values and they seem to be correct

![image](https://github.com/etherspot/skandha/assets/7760339/cd463c72-a906-44f8-81a6-e9480b1bd238)

That's 0.024 Gwei